src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
```

Comment:
that what I see when I press enter, use link from previous task
![image](https://user-images.githubusercontent.com/37161315/167623404-1886f254-603e-451f-a6d2-41cc700ef560.png)